core-library/src/main/java/org/silverpeas/core/contribution/contentcontainer/content/ContentManager.java

```diff
@@ -126,11 +118,7 @@
     }
   }
 
-  // Container manager
-  private ContainerManager m_containerManager = null;
-
   public ContentManager() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1186)